src/main/java/org/broadinstitute/hellbender/tools/exome/ExomeReadCounts.java

```diff
@@ -289,7 +295,8 @@ private PrintWriter openOutputWriter(final File output, final String headerText)
 
     @Override
     public void apply(final SAMRecord read, final ReferenceContext referenceContext, final FeatureContext featureContext) {
-        final GenomeLoc readLocation = genomeLocFactory.createGenomeLoc(read);
+
+        final SimpleInterval readLocation = new SimpleInterval(read.getContig(),read.getStart(),read.getEnd());
```

Comment:
You shouldn't have to do this.  Could you add this to `SimpleInterval` as a copy constructor from `Locatable`